### DISCUSSION TITLE
Log missing message key/path when building?

### DISCUSSION THREAD

**Discussion by jonasmerlin** [amannn/next-intl#1221] [General]
So when I'm building my app using `npm run build` I'm getting the following error 3 times:

```
s [Error]: MISSING_MESSAGE
    at ...,
  code: 'MISSING_MESSAGE',
  originalMessage: undefined
}
```

I already defined custom `onError` as well as `getMessageFallback` handlers. But in this case, `getMessageFallback` doesn't seem to be called. So I want to log the missing message key inside `onError`, so that I can add it. How can I do this?

I'm on next version 14.0.1 and next-intl version 3.17.2 btw.

**Comment by amannn** [comment]
Have you configured `onError`/`getMessageFallback` in `i18n.ts` but not on `NextIntlClientProvider`? 

If in doubt, please review [the error handling docs](https://next-intl-docs.vercel.app/docs/usage/configuration#error-handling) (esp. the last paragraph when you have `i18n.ts` selected). Note that I've added https://github.com/amannn/next-intl/commit/88dfb10ee3d20745bd61426c9cc1427b1d0da406 which might provide more clarity for this case.

As a side note, I know the situation with having to forward `onError` manually is far from ideal, I'm planning to simplify this as part of https://github.com/amannn/next-intl/issues/779.
